api/managementpb/checks.proto

```diff
@@ -28,6 +29,38 @@ message SecurityCheckResult {
   string service_name = 6;
 }
 
+// CheckResultSummary is a summary of check results.
+message CheckResultSummary {
+  string service_name = 1;
+  string service_id = 2;
+  // Number of failed checks for this service that are critical
+  uint32 critical_count = 3;
+  // Number of failed checks for this service that are major
+  uint32 major_count = 4;
+  // Number of failed checks for this service that are trivial
+  uint32 trivial_count = 5;
+}
+
+// CheckResult represents the check results for a given service.
+message CheckResult {
+  string summary = 1;
+  string description = 2;
+  Severity severity = 3;
+  map<string, string> labels = 4;
+  // URL containing information on how to resolve an issue detected by an STT check.
+  string read_more_url = 5;
+  // Name of the monitored service on which the check ran.
+  string service_name = 6;
+  // ID of the monitored service on which the check ran.
+  string service_id = 7;
+  // Name of the check that failed
+  string check_name = 8;
+  // ID of the check result as stored in AlertManager
+  string alert_id = 9;
+  // Silence status of the check result
+  bool silenced = 10;
+}
```

Comment:
We may add `check_type` field (would be something like `MONGODB_REPLSETGETSTATUS`) or split it into `check_type` and `db_family` as requested in the tkt.

Reply:
Yeah, but I don't think `db_family` is currently supported for checks v1.

Reply:
I think the field may be empty if there is no data to fill it. What do you think?

Reply:
I see, that could work then. but right now it will *always* be empty so I thought maybe we can pend it until we have proper support for it across platform/managed.